src/main/antlr/com/sleekbyte/tailor/antlr/Swift.g4

```diff
@@ -666,6 +672,10 @@ postfixExpression
  | postfixExpression '?'                                         # optionalChainingExpression
  ;
 
+// GRAMMAR OF AN ARGUMENT NAME
+argumentNames : argumentName argumentNames?  ;
+argumentName: (identifier | '_') ':'?  ; // Swift Language Reference has argumentName → identifier :
```

Comment:
Is the `':'` supposed to be optional? (another deviation from the Swift Language Reference)